doc/metric_spec.md

```diff
@@ -102,6 +102,12 @@ The value can vary from 0, 1 , 5 etc to `+Inf`, that  correspond to the infinity
 The number of migration threshold pro node and resource ID set by a pacemaker cluster. 
 Possible values are positive numbers.
 
+### `ha_cluster_pacemaker_resource_configuration_changes`
```

Comment:
I would suggest changing the name from `resource_configuration_changes` to `last_configuration_change` or `last_cib_change`, as this is not really resource-related, but it's cluster-wide.

Reply:
I agree. Something like `ha_cluster_pacemaker_last_config_change`.

Reply:
done thx

Reply:
Doesn't real matter, or `ha_cluster_pacemaker_config_last_change` to be object-oriented :-)